src/main/java/me/gaagjescraft/network/team/manhunt/utils/config/icon/IconConfig.java

```diff
@@ -0,0 +1,42 @@
+package me.gaagjescraft.network.team.manhunt.utils.config.icon;
+
+import org.bukkit.configuration.file.FileConfiguration;
+
+import java.util.List;
+
+public class IconConfig {
+    private final String material;
+    private final String displayName;
+    private final List<String> lore;
+
+    public IconConfig(String material, String displayName, List<String> lore) {
+        this.material = material;
+        this.displayName = displayName;
+        this.lore = lore;
+    }
+
+    public String getMaterial() {
+        return material;
+    }
+
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    public List<String> getLore() {
+        return lore;
+    }
+
+    public static IconConfig load(String prefix, FileConfiguration fileConfiguration) {
+        return new IconConfig(
+                fileConfiguration.getString(prefix + "-material"),
```

Comment:
why not use prefix + ".material" as to use sub sections in the config?

Reply:
I guess i can do that but it will require config migrations (current keys for icons as far as i remember are ...-material, ...-displayname and ...-lore)